src/ipv4.c

```diff
@@ -279,6 +279,9 @@ int ipv4_protect_tunnel_route(struct tunnel *tunnel)
 
 err_destroy:
 	route_destroy(def_rt);
+	tunnel->ipv4.route_to_vpn_is_added = 0;
+	log_warn("Protecting tunnel route is fail. \
+But this can be working except some case.\n");
 
```

Comment:
I'd suggest an improvement of the language here: 
"Protecting tunnel route has failed. \
But this can be working except for some cases."